apps/stupidchat/stupidchat.go

```diff
@@ -0,0 +1,25 @@
+// Package stupidchat provides details for the Stupid Chat applet.
+package stupidchat
+
+import (
+	_ "embed"
+
+	"tidbyt.dev/community/apps/manifest"
+)
+
+//go:embed stupid_chat.star
+var source []byte
+
+// New creates a new instance of the Stupid Chat applet.
+func New() manifest.Manifest {
+	return manifest.Manifest{
+		ID:          "stupid-chat",
+		Name:        "Stupid Chat",
+		Author:      "harrisonpage",
+		Summary:     "Tidbyt Messaging",
+		Desc:        "Send messages to your Tidbyt via https://stupid.chat or an API.",
```

Comment:
Could you also add a note in the description highlighting that the Stupid Chat service is external and not run by Tidbyt? You can see a similar example in the "Textbyt" app.